src/plays/typing-speed-test/components/Stats.js

```diff
@@ -0,0 +1,25 @@
+const Stats = ({ wpm, cpm, accuracy }) => {

```

Comment:
Same Here we can pass the one stats object which will contain this information.

```js
const stats = { wpm, cpm, accuracy }
```

Reply:
Done